.devcontainer/devcontainer.json

```diff
@@ -22,10 +22,38 @@
                 "ms-azuretools.azure-dev",
                 "ms-azuretools.vscode-bicep",
                 "ms-azuretools.vscode-docker",
-                "ms-vscode.js-debug"            ]
+                "ms-vscode.js-debug"],
+				"settings": {
+					"remote.autoForwardPorts": false
```

Comment:
By disabling this setting, it removes a confusing auto-forward prompt in Codespaces that offers to open the auto-forwarded port/URL in the browser (always shows the wrong port, not the one we want for the dashboard).